projects/solar/src/relay_fsm.c

```diff
@@ -28,35 +31,38 @@ static void prv_assert_relay(SoftTimerId timer_id, void *context) {
 
   data_store_get_is_set(DATA_POINT_CURRENT, &is_set);
 
+  // Checking if the current will ever be set
   if (!is_set) {
-    if (((RelayFsmStorage *)context)->is_set_counter >= MAX_NUMBER_OF_CURRENT_CHECKS) {
+    if (storage->is_set_counter >= MAX_NUMBER_OF_CURRENT_CHECKS) {
       LOG_DEBUG("Aborting, The current is not set\n");
-      fault_handler_raise_fault(EE_SOLAR_RELAY_OPEN_ERROR, 0xFF);
+      fault_handler_raise_fault(EE_RELAY_ERROR_CURRENT_NEVER_SET, 0);
```

Comment:
What I meant is that we should still have the `EE_SOLAR_RELAY_OPEN_ERROR` as the first argument here (the fault ID), but as the fault data (the second argunent) we should have `EE_RELAY_ERROR_CURRENT_NEVER_SET`.

Reply:
done